css/store.css

```diff
@@ -0,0 +1,196 @@
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+/**
+ * Add your custom styles below
+ * 
+ * Remember: 
+ * - Be organised, use comments and separate your styles into meaningful chunks
+ *    for example: General styles, Navigation styles, Hero styles, Footer etc.
+ * 
+ * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
+ */
+
+#navbar-button {
+  margin-left: 25px;
+  margin-right: 25px;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 10px;
+}
+
+.navbar-logo {
+  display: flex;
+  margin: auto;
+  width: 2%;
+  margin-left: 200px;
+}
+
+.navbar-list {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  list-style-type: none;
+  font-size: smaller;
+  margin-right: 60px;
+}
+a:hover {
+  color: red;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+.main-section {
+  display: flex;
+  flex-direction: row;
+}
+
+h1 {
+  color: orange;
+  font-weight: 100;
+  font-size: 30px;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 50%;
+  margin-right: 2%;
+  margin-left: 3%;
+}
+
+.name {
+  display: flex;
+  flex-direction: row;
+}
+
+.field {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  margin-right: 20px;
+}
+
+.address {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-top: 5%;
+  margin-bottom: 5%;
+}
+
+.field-2 {
+  display: flex;
+  flex-direction: column;
+  margin-right: 20%;
+  margin-top: 5%;
+}
+
+.city {
+  display: flex;
+  flex-direction: row;
+}
+
+.field-3 {
+  display: flex;
+  flex-direction: column;
+  width: 60%;
+}
+
+.field-4 {
+  display: flex;
+  flex-direction: column;
+  width: 40%;
+  margin-left: 5%;
+}
+
+h2 {
```

Comment:
Better to target a class rather than every `h2`, so that unwanted styles aren't applied to every `h2`